cornice/cors.py

```diff
@@ -125,7 +125,7 @@ def apply_cors_post_request(service, request, response):
             'Access-Control-Allow-Credentials' not in response.headers):
         response.headers['Access-Control-Allow-Credentials'] = 'true'
 
-    if request.method is not 'OPTIONS':
+    if request.method != 'OPTIONS':
```

Comment:
I cannot reproduce with my shell, on python 2.6.9 and python 2.7.6.

Do you have more information about that?

```python
>>> method = "OPTIONS"
>>> method is not "OPTIONS"
False
>>> method = "NOT OPTIONS"
>>> method is not "OPTIONS"
True
```